cg/constants/delivery.py

```diff
@@ -1,62 +1,74 @@
 """Constants for delivery"""
 
+
 BALSAMIC_ANALYSIS_CASE_TAGS = [
-    "clinical-vcf-pass",
-    "clinical-vcf-pass-index",
-    "manta",
-    "cnvkit",
-    "cnv-gene-metrics",
-    "cnv-scatter",
-    "cnv-diagram",
-    "multiqc-html",
-    "tumor-bam",
-    "tumor-bam-index",
-    "tumor-cram",
-    "tumor-cram-index",
-    "normal-bam",
-    "normal-bam-index",
-    "normal-cram",
-    "normal-cram-index",
+    {"cnv", "cnvkit", "vcf-all", "annotated-somatic-vcf-all"},
+    {"cnv", "cnvkit", "vcf-all", "annotated-somatic-vcf-all-index"},
+    {"vardict", "snv", "annotated-somatic-vcf-pass, vcf-pass"},
+    {"vardict", "snv", "annotated-somatic-vcf-pass-index", "vcf-pass"},
+    {"manta", "sv", "annotated-somatic-vcf-pass", "vcf-pass"},
+    {"manta", "sv", "annotated-somatic-vcf-pass-index", "vcf-pass"},
+    {"cnv-scatter"},
+    {"cnv-diagram"},
+    {"multiqc-html"},
+    {"balsamic-report"},
+    {"normal-cram"},
+    {"normal-cram-index"},
+    {"tumor-cram"},
+    {"tumor-cram-index"},
```

Comment:
We don't have CRAM for WGS yet, but we can deliver BAM/CRAM when customer asks for it.

Reason for including 3 "snv" deliveries is WGS Tumor only, WGS Tumor Normal, and WES/TGA Tumor-only/Tumor-Normal. I'll add this to the issue as well.

This should cover bare minimum until we have unique tags in place in future:

```suggestion
    {"cnv", "cnvkit", "vcf-all", "annotated-somatic-vcf-all"},
    {"cnv", "cnvkit", "vcf-all", "annotated-somatic-vcf-all-index"},
    {"snv", "annotated-somatic-vcf-pass, vcf-pass", "vardict"},
    {"snv", "annotated-somatic-vcf-pass-index", "vcf-pass", "vardict"},
    {"snv", "annotated-somatic-vcf-pass", "vcf-pass", "tnscope"},
    {"snv", "annotated-somatic-vcf-pass-index", "vcf-pass", "tnscope"},
    {"snv", "annotated-somatic-vcf-pass", "vcf-pass", "tnhaplotyper"},
    {"snv", "annotated-somatic-vcf-pass-index", "vcf-pass", "tnhaplotyper"},
    {"sv", "annotated-somatic-vcf-pass", "vcf-pass", "manta"},
    {"sv", "annotated-somatic-vcf-pass-index", "vcf-pass", "manta"},
    {"cnv-scatter"},
    {"cnv-diagram"},
    {"multiqc-html"},
    {"balsamic-report"},
    {"normal-cram"},
    {"normal-cram-index"},
    {"tumor-cram"},
    {"tumor-cram-index"},
```

Reply:
Great I'll add this